### PR TITLE
Front page rework

### LINKED ISSUE #117: Replace intro/excerpt with a bit of social proof?
The intro/excerpt below the image is redundant and a bit empty. It'd be good if we could replace it with some kind of social proof.  Something like this...

> ODK has helped tens of thousands of organizations collect billions of forms and surveys. Users include Google, New York City, Red Cross, Kiva, Carter Center, USAID, WHO, CDC, Jane Goodall Institute, and many others. From election monitoring to tracking outbreaks, ODK tools are the de-facto standard for data collection.

### PR DESCRIPTION
**Pull request by yanokwa**
Closes #117 

#### What is included in this PR?
In my review of #128, I kept making a lot of changes, so I figured I'd just do it in a separate PR. My changes are...
* Flashier title that isn't so dependent on attributes that we describe later. "Standard" is also a nice play on mission and values.
* Tens of thousands feels like a safe number.
* Sync up call to action with the call at the bottom, 
* Order social proof by big names first, then some of the physically bigger names. Reads a little easier. I try to link to folks who are still using the software and external (e.g., not Nafundi) parties.
* Instead of focusing on building, focusing on what the tools enable.
* Tweak language to plug both turnkey and customization options
* Hopefully catchier language around intuitiveness and scalablility
* Change picture positions to make them match the wording.
* Oh, and I say "mobile data collection" because that's what it is and that's what people search for.

screen shot 2018-05-28 at 16 56 30

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
...vaccination-activity-in-banadir-and-lower-and-middle-shabelle-regions.html), [CDC](https://www.cdc.gov/ophss/chiic/projects/2014/ideas/using-open-data-kit-odk-for-rapid-data-collection-transport-visualization-and-management-of-malaria-data-in-siaya-county-kenya.html), [USAID](https://www.agrilinks.org/blog/partnership-open-source-technology-opens-door-improved-survey-data), the [Red Cross and Red Crescent](http://www.ifrc.org/en/news-and-media/news-stories/americas/bolivia/bolivian-red-cross-and-ifrc-visits-border-between-bolivia-and-brazil--72334/), the [Carter Center](http://blog.cartercenter.org/2012/08/06/carter-center-on-leading-edge-of-technology-use-in-election-observation/), the [Jane Goodall Institute](http://www.janegoodall.org/2016/11/21/monitoring-forests-2/) and [many others](https://forum.opendatakit.org/c/showcase). <br><br> From [monitoring important elections](http://democracyinternational.com/resources/2017-final-monitoring-report-of-the-albanian-assembly-elections) to [tracking disease outbreaks](https://news.cs.washington.edu/2014/11/18/paul-g-allen-tackles-ebola-with-10000-odk-equipped-smartphones/),<br>Open Data Kit is the standard for mobile data collection."
```
Comment (yanokwa) [inline index.html]: s/tens of thousands of organizations/millions of people/

Sounds better and easier to prove :)

**Comment by jbeorse** [comment]
Looks like Adam has incorporated these changes in his open PR #128 though the wording is slightly different: 

- s/help tens of thousands/are helping thousands/
- s/collect, manage, and use their data/collect data through forms and surveys/
- s/mobile data collection/in-the-field data collection/
- Some of the links are different

These differences seem fairly innocuous to me. I just wanted to point it out so nothing gets lost in the merge.

**Comment by adammichaelwood** [comment]
I think all of this is an improvement over my PR.

File: index.html [hunk 2]
```diff
@@ -1,36 +1,36 @@
...
-    alt: "Build data collection forms using powerful tools."
-    title: "Build data collection forms using powerful tools."
-    excerpt: "Create forms to collect critical information using our easy-to-use, drag-and-drop design tools. From simple Q&A to powerful logic flows, we've got you covered."
-  - image_path: /assets/images/visualize.png
-    alt: "Collect, upload, aggregate, & extract data."
-    title: "Collect, upload, aggregate, & extract data."
-    excerpt: "Use our mobile applications on tablets or phones to easily collect responses like text, locations, and media. Then manage, visualize, and export that data using our tools."
+    alt: "Collect the data you need to make decisions"
+    title: "Collect the data you need to make decisions"
+    excerpt: "From text to pictures to location, collect the data you need to make the right decisions. Our tools also support complex branching, multiple languages, and work offline!"
```
Comment (adammichaelwood) [inline index.html]: The syntax here is a bit weird to me.

I think what you are trying to say is...
We support:
 - complex branching
 - multiple languages
 - work offline

**Pull request merged**